cli/command/swarm/leave.go

```diff
@@ -19,15 +19,15 @@ func newLeaveCommand(dockerCli *command.DockerCli) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "leave [OPTIONS]",
-		Short: "Leave a swarm",
+		Short: "When run from a worker, leave the swarm.",
```

Comment:
I'd prefer to make this more succinct. For context, here's the relevant part of the output of `docker swarm --help` in 1.12:

```
Commands:
  init        Initialize a swarm
  join        Join a swarm as a node and/or manager
  join-token  Manage join tokens
  update      Update the swarm
  leave       Leave a swarm
```

I think `Leave the swarm` is fine, but if you feel strongly about noting that it can only be run from a worker, how about `Leave the swarm (workers only)`?